test/src/main/java/org/apache/accumulo/test/NamespacesIT.java

```diff
@@ -755,6 +755,7 @@ public void testPermissions() throws Exception {
 
       loginAs(root);
       c.securityOperations().grantNamespacePermission(u1, n1, NamespacePermission.ALTER_TABLE);
+      c.securityOperations().grantTablePermission(u1, t3, TablePermission.READ);
       loginAs(user1);
       user1Con.tableOperations().setProperty(t3, Property.TABLE_FILE_MAX.getKey(), "42");
```

Comment:
Alter table permissions should be sufficient for reading properties. It should not require READ permission. I'm also wondering if READ is even the right permission at all. Should users who merely have permission to scan a table be presumed to be able to read the configuration for that table? Or should they have `ALTER_TABLE` permission instead.

Reply:
SystemPermission.SYSTEM is the equivalent to the ALTER versions, so that makes sense.